### DISCUSSION TITLE
I paid for BitKit v1 and did not receive it

### DISCUSSION THREAD

**Discussion by alaaelhawy1975** [community/community#129863] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

https://www.oklink.com/eth/tx/0xa304e49cc982eb39e39b5c657b209284d6159b529026c9792d64abe36de1af95

**Comment by alaaelhawy1975** [comment]
![image](https://github.com/community/community/assets/173929358/8b76281e-5d07-48bd-9a5b-7c6d6585b3d6)

**Comment by ebndev** [comment]
Hi @alaaelhawy1975, thanks for being a part of the GitHub Community!

If you're looking for help with this, you might want to try contacting their support instead of GitHub.

We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
